src/pay/payment_producer.py

```diff
@@ -122,6 +125,15 @@ def run(self):
             time.sleep(5)
 
             try:
+
+                # Check if local node is bootstrapped; sleep if needed; restart loop
+                if not self.node_is_bootstrapped():
+                    logger.info("Local node, {}, is not in sync with the Tezos network. Will sleep for {} blocks and check again."
+                                .format(self.node_url, BOOTSTRAP_SLEEP))
+                    self.wait_until_next_cycle(BOOTSTRAP_SLEEP)
```

Comment:
Should this not be `self.wait_for_blocks(BOOTSTRAP_SLEEP)` instead?

Reply:
Yes it should